CONTRIBUTING.rst

```diff
@@ -333,6 +332,60 @@ Recommendation for the Pull Requests:
    -  If applicable, ensure that your fix includes appropriate tests. Adding tests for your bug fix helps prevent future regressions and maintains the stability of the software.
 
 
+Contributing to test data
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your changes include updating test data, you may work with the ``gpm/tests/data/`` directory as you would with any other git repository.
+You need to ask the maintainers to become a contributor on the `gpm_api_test_data <https://github.com/ghiggi/gpm_api_test_data>`_ repository to create a branch and a Pull Request.
+The GPM-API repository keeps track of the currently checked-out commit of the test-data repository. When the checked-out commit changes, you can register this change in the GPM-API repository by running
+
+.. code-block:: bash
+
+    git add gpm/tests/data
+
+and committing.
+
+
+To submit your contribution that involves modifying test data, please follow this procedure.
+
+(A: GPM-API repository, B: test-data repository)
+
+1. Make a *feature branch* for B
+
+.. code-block:: bash
+
+    cd gpm/tests/data
+    # Inside this directory, following git commands will apply to B
+    git checkout -b my-feature-branch
+    ...
+
+2. Have A point to the *feature branch* of B
+
+.. code-block:: bash
+
+    # From the root of the GPM-API repository
+    git add gpm/tests/data
+    git commit
+    ...
+
+3. Make two PRs (for A and B) and get both accepted
+4. Have the B’s PR merged into the B's *main branch*
+5. Update A to point to B’s updated *main branch* (instead of the old *feature branch*)
+
+.. code-block:: bash
+
+    # Checkout the main branch of the test-data repository
+    cd gpm/tests/data
+    git checkout main
+    git pull
+
+    cd ../../..
```

Comment:
Maybe not necessary? /.../.../

Reply:
`cd ...` does not work in bash, it does in zsh